README.md

```diff
@@ -40,15 +43,42 @@ It’s a Rails app that:
     $ bundle install --without production
     ```
 
-7. Set up your database. The simple way to do this is:
+7. Set up your database
 
+    Make a local copy of the database configuration template file:
+    
     ```sh
-    $ bundle exec rake db:setup
+    cp config/database.yml.example config/database.yml
+    ```
+
+    First create users, databases and configure extensions:
```

Comment:
I don’t think this should be recommended as a default configuration. In the vast majority of cases, there’s no reason anyone needs an extra password on their dev database. It’s great that we can support that, but it’s not necessary and we shouldn’t burden basic setup with a lot of unnecessary steps.

If a user wants to password-protect their local DB, we should cover that in the “advanced setup” section.